feat(curriculum): add animations to block 3.2 of the English Curriculum (#54033)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-talk-about-hobbies-and-interests/657e758778ff8dc01567c9e7.md

title: Task 27
challengeType: 19
dashedName: task-27
---

<!-- (Audio) Jake: That's great! How long have you been into cycling? -->

# --description--

> Remember, to form questions with the Present Perfect Tense, you need to rearrange the sentence structure. This involves changing the position of the subject and the auxiliary verb `have`. The verb in the participle form (in this case, `been`) goes after the person. For example: `How long have you been a doctor?`

Reviewer: ```suggestion
Remember, to form questions in the Present Perfect Tense, the sentence structure must be rearranged by changing the position of the subject and the auxiliary verb `have`. The verb in its participle form (in this case, `been`) is placed after the subject. For example: `How long have you been a doctor?`
```
